deploy/elasticsearch/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
 # bitnami/elasticsearch
 # docker pull bitnami/elasticsearch  
   registry: docker.io
-  repository: apecloud/elasticsearch
+  repository: bitnami/elasticsearch
```

Comment:
The image repository has been upgraded to apecloud/elasticsearch, please revert this back.